source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -54,6 +54,9 @@ class AuthenticatorImpl : public Logger::Loggable<Logger::Id::jwt>,
 
   TimeSource& timeSource() { return time_source_; }
 
+  // convert double to string
+  static std::string convertDoubleToString(double double_value);
```

Comment:
Make this as global static function,  not a static class function.  so no need to declare in the class